pyspecProcScripts/load_data.py

```diff
@@ -462,16 +217,40 @@ def proc_nutation_chunked(s, fl=None):
     return s
 
 
+def proc_nutation_v2(s, fl=None):
+    logging.debug("loading pre-processing for nutation")
+    s.set_units("indirect", "s")
+    s.ft(["ph1"])
+    s.set_prop("coherence_pathway", {"ph1": 1})
+    s.set_units("t2", "s")
+    s["t2"] -= s.get_prop("acq_params")["tau_us"] * 1e-6
+    s *= s.shape["nScans"]
+    s.squeeze()
+    s.reorder(["ph1"]).set_units("t2", "s")
+    if fl is not None:
+        fl.next("Raw Data - Time Domain")
+        fl.image(s.human_units())
+    s.ft("t2", shift=True)
+    if fl is not None:
+        fl.next("Raw Data- Frequency Domain")
+        fl.image(s)
+    return s
+
+
 def proc_nutation_v4(s, fl=None):
     if s.shape["indirect"] > s.shape["nScans"]:
         s.reorder(["ph1", "nScans", "indirect"])
     else:
         s.reorder(["ph1", "indirect", "nScans"])
     s["indirect"] *= 1e-6  # why is it labeled in μs??
     s.rename("indirect", "p_90")
+    s.set_prop("coherence_pathway", {"ph1": 1})
```

Comment:
at some point, don't we start storing this in the file at the time of acquisition? since this is version 4, can you check if the version 4 code is yet doing that?